packages/web/src/components/ui/Dropdown/Dropdown.tsx

```diff
@@ -1,72 +1,72 @@
-/* eslint-disable react/button-has-type */
-import React, { useState, MouseEvent, MouseEventHandler } from 'react';
-import clsx from 'clsx';
+import React, { useRef, useState } from 'react';
 import styles from './Dropdown.module.scss';
+import clsx from 'clsx';
+import Button from '../Button/Button';
+import { useOnClickOutside } from 'hooks/useOutsideAlerter';
+import { Icon } from '../Icon/Icon';
+import Badge from '../Badge/Badge';
 
-export type Props = {
+type Props = {
+  triggerText?: string;
```

Comment:
This could be safely changed to `triggerElement?: React.ReactNode` to become more unviersal.